src/components/leaflet/LeafletMap.jsx

```diff
@@ -2,6 +2,7 @@ import { MapContainer, TileLayer, Marker } from 'react-leaflet';
 import style from './leaflet.module.scss';
 import { Icon } from 'leaflet';
 import MarkerIcon from '../../assets/images/icon-location.svg';
+import { MyComponent } from '../../helpers/myComponent';
```

Comment:
Is there a better name for the component?

Reply:
done